extension.js

```diff
@@ -21,31 +21,31 @@ function enable() {
      * Listen to enabled extension, if Dash to Dock is on the list or become active,
      * we disable this dock.
      */
-    dockManager=null; // even if declared, we need to initialize it to not trigger a referenceError.
```

Comment:
Are you sure this isn't needed anymore?

Reply:
Yes, I don't get any warning (I tested it with and without dash-to-dock enabled).